### ISSUE TITLE
error while trying to install nix on macOS 15 <autogenerated-issue>

### ISSUE THREAD

**Issue by LATBauerdick** [DeterminateSystems/nix-installer#1029]
## Error
```
Error: 
   0: Multiple errors
```

## Metadata
|key|value|
|--|--|
|**version**|0.19.0|
|**os**|macos|
|**arch**|aarch64|

**Comment by cole-h** [comment]
Usually when there are "Multiple errors", they should all show up in the list of errors... Was that not the case for you?

However, going off of the "macOS 15" you put in the title, it sounds like this may be related to https://github.com/DeterminateSystems/nix-installer/issues/1001. There are workarounds in that issue.

Going to close this as a dupe of that one, but if the errors were unrelated, feel free to reopen!

**Comment by LATBauerdick** [comment]
thank you for the prompt response. The problem must have to do with that I did not delete but merely commented out the line in `fstab` that mounted a previous version of the nix store on `/nix`. Also, somehow nixbld1 was still there from my previous installation before upgrading to the Sequoia beta, which also confused the install process. I deleted those user entries, and eventually everything worked, and Nix is running just fine on my upgraded system. Thanks again for all your support!
